backend/src/app/domain/components/services.py

```diff
@@ -169,160 +169,112 @@ async def delete_many(
             ),
         )
 
+    async def _create_element(
+        self,
+        element: ModelDictT[ComponentElement],
+        component_id: str,
+        processes_service,
+    ):
+        element_dict = (
+            element.to_dict() if type(element) is ComponentElement else element
```

Comment:
It would probably be better to use `isinstance`, but I’d rather do that upstream if possible anyway, so that we can be more assertive on the type entering this method.